<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GG::EditDialog</class>
 <widget class="QDialog" name="GG::EditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>409</width>
    <height>325</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <property name="margin">
    <number>6</number>
   </property>
   <item row="3" column="0" colspan="2">
    <widget class="QGroupBox" name="serversGroupBox">
     <property name="title">
      <string>Servers</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <property name="topMargin">
       <number>6</number>
      </property>
      <item row="0" column="0">
       <widget class="QLabel" name="serversLabel">
        <property name="text">
         <string>If connection to HUB fails, we will try to connect to one of these servers.</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QPlainTextEdit" name="serversTextEdit"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="uinLabel">
     <property name="text">
      <string>UIN:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="uinEdit"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="passwordLabel">
     <property name="text">
      <string>Password:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="passwordEdit">
     <property name="echoMode">
      <enum>QLineEdit::Password</enum>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="2">
    <widget class="QGroupBox" name="settingsGroupBox">
     <property name="title">
      <string>Settings</string>
     </property>
     <layout class="QFormLayout" name="formLayout">
      <item row="0" column="0">
       <widget class="QCheckBox" name="SSLCheckBox">
        <property name="text">
         <string>Use secure connection</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QComboBox" name="startupComboBox">
        <item>
         <property name="text">
          <string>Previous</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Available</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Be right back</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Do not disturb</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Free for chat</string>
         </property>
        </item>
        <item>
         <property name="text">
          <string>Invisible</string>
         </property>
        </item>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="startupLabel">
        <property name="text">
         <string>Startup status:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QCheckBox" name="friendsCheckBox">
        <property name="text">
         <string>Status only visible to friends</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>uinEdit</tabstop>
  <tabstop>passwordEdit</tabstop>
  <tabstop>SSLCheckBox</tabstop>
  <tabstop>friendsCheckBox</tabstop>
  <tabstop>startupComboBox</tabstop>
  <tabstop>serversTextEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>GG::EditDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>407</x>
     <y>269</y>
    </hint>
    <hint type="destinationlabel">
     <x>359</x>
     <y>25</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
